src/html/index.html

```diff
@@ -11,22 +11,26 @@ <h3 data-module="example">{{ message }}</h3>
       <div style="background-color:#4a4a4a; padding:10px; height:auto">
         <button class="c-btn  c-btn--primary">Register</button>
         <button class="c-btn  c-btn--icon  c-btn--icon-huge  c-btn--icon_darkbg">{{sprite("play")}}</button>
-        <button class="o-flex  c-btn  c-btn--icon  c-btn--icon-small  c-btn--icon_darkbg">{{sprite("play")}}
-          <span class="o-flex--text">watch the video</span>
-        </button>
+
+        <div class="c-watch">
+          <button class="c-watch--btn  c-btn  c-btn--icon  c-btn--icon-small  c-btn--icon_darkbg">{{sprite("play")}}
+            <span class="c-watch--text">watch the video</span>
+          </button>
+        </div>
       </div>
 
       <div style="background-color:#fbfbfb; padding:10px; height:auto">
         <button class="c-btn  c-btn--secondary">Register</button>
         <button class="c-btn  c-btn--icon  c-btn--icon-huge  c-btn--icon_lightbg">{{sprite("play")}}</button>
 
-
         <div class="c-share">
```

Comment:
Let's move `.c-share` into a shared .html file so we can re-use it throughout the page.